ach/LC_MESSAGES/django.po

```diff
@@ -7,16 +7,16 @@ msgstr ""
 "POT-Creation-Date: 2022-09-06 23:52-0700\n"
 "PO-Revision-Date: 2020-07-29 07:07+0000\n"
 "Last-Translator: denish <denish@mozilla-uganda.org>\n"
-"Language-Team: LANGUAGE <LL@li.org>\n"
```

Comment:
I believe we can drop all changes in this file completely

Reply:
I looked at this yesterday - I believe it got linted and/or refactored.  All the changes are line breaks. It's not a supported SUMO language at the moment anyway so I think it could stay or go without much impact. I'd vote for it leaving it in the PR since the file was, hopefully, improved.

Reply:
If you prefer to keep it let's move it in a separate PR and keep this PR relevant to headers.